CMakeLists.txt

```diff
@@ -182,7 +187,7 @@ set(CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION ${CMAKE_INSTALL_FULL_MANDIR}
 install(FILES
           "include/sys/event.h"
         DESTINATION
-          "${CMAKE_INSTALL_FULL_INCLUDEDIR}/sys"
+          "${CMAKE_INSTALL_FULL_INCLUDEDIR}/kqueue/sys"
```

Comment:
Is this already done in the official debian packages? I guess consumers of libkqueue should be using pkg-config to get the include path so it shouldn't matter where the headers are located.

Reply:
ah and the current .pc appends kqueue anyway